src/dataflow/src/decode/protobuf.rs

```diff
@@ -21,12 +21,15 @@ pub struct ProtobufDecoderState {
 impl ProtobufDecoderState {
     pub fn new(
         ProtobufEncoding {
-            descriptors,
-            message_name,
+            descriptors:
+                EncodedDescriptors {
+                    file_descriptor_set,
+                    message_name,
+                },
             schema_registry_config,
         }: ProtobufEncoding,
     ) -> Result<Self, anyhow::Error> {
-        let descriptors = DecodedDescriptors::from_bytes(&descriptors, message_name)
+        let descriptors = DecodedDescriptors::from_bytes(&file_descriptor_set, message_name)
```

Comment:
Yeah based on the other use of `DecodedDescriptors::from_bytes`, it seems we could just `impl From<EncodedDescriptors> for DecodedDescriptors`?  Unless that causes us to hit the layering / dependency violations